docs/_components/spinner.md

```diff
@@ -13,7 +13,15 @@ Spinners can be helpful to signal to a user that the current process may require
 {% capture example %}
 <div class="lg-spinner">
   <div>
-    <img src="{{ site.baseurl }}/assets/img/spinner.gif" srcset="{{ site.baseurl }}/assets/img/spinner@2x.gif" width="144" height="144" class="text-middle" alt="Spinner" />
+    <img
+      src="{{ site.baseurl }}/assets/img/spinner.gif"
+      srcset="{{ site.baseurl }}/assets/img/spinner.gif,
+              {{ site.baseurl }}/assets/img/spinner@2x.gif 2x"
+      width="144"
+      height="144"
+      class="text-middle"
+      alt="Spinner"
```

Comment:
Not really relevant to the intent of this pull request, but in the course of applying the changes for f7591b6, it struck me whether or not "Spinner" is a good alternative text.

- Is a spinner [decorative](https://www.w3.org/WAI/tutorials/images/decorative/)?
- Or is this a [live region](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-live) which is [busy](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-busy)?
   - https://www.smashingmagazine.com/2015/04/its-alive-apps-that-feed-back-accessibly/
   - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
- Or maybe "Spinner" is a pragmatic solution that will be reasonably well-understood?

Reply:
I think that it's decorative, but we also have an entirely separate page on our accessibility policy and alt text, so I wouldn't worry too much here: https://design.login.gov/accessibility/policies/